oop/task_4.py

```diff
@@ -0,0 +1,18 @@
+class Vehicle:
+    def __init__(self, make, model, year):
+        self.make = make      
+        self.model = model    
+        self.year = year     
+
+class ElectricCar(Vehicle):
+    def __init__(self, make, model, year, battery_size):
+    
+        super().__init__(make, model, year)
+        self.battery_size = battery_size  # Дополнительный атрибут для электромобиля
```

Comment:
Check what happens if we do not use `super` method in `init`

```suggestion
class Vehicle:
    def __init__(self, make, model, year):
        self.make = make
        self.model = model
        self.year = year
       
class ElectricCar(Vehicle):
    def __init__(self, battery_size):
        self.battery_size = battery_size
        print(f"Battery Size: {battery_size}")
```